synapse/lib/stormlib/storm.py

```diff
@@ -64,3 +65,82 @@ async def _evalStorm(self, text, cast=None):
             valu, _ = casttype.norm(valu)
 
         return valu
+
+class StormPoolSetCmd(s_storm.Cmd):
+    '''
+    Setup a Storm query offload mirror pool for the Cortex.
+    '''
+    name = 'storm.pool.set'
```

Comment:
Since we're adding additional cortex commands do we want to namespace them like `cortex.storm.pool.set`, in a fashion similar to how we namespaced the httpapi commands?

Reply:
I could see it belonging there, but we do also have `$lib.storm` already as well. Would there likely be other commands we would want to add to either namespace where it would make sense to prefer one over the other?